botbuilder-contrib/botbuilder-storage-mongodb
# Separate file per class

## Purpose


* Break apart monolith file into several

## Does this introduce a breaking change?



```
[ ] Yes
[x] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[x] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other - Please describe:
            _______________
```

## How to Test

* Get the code
* Observe several .ts files instead of just one.

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code



```
```

## What to Check

Verify that the following are valid

* Several `.ts` files instead of one
* May require changes to import for framework builders, but not clients.

## Other Information
